src/cli/main.cpp

```diff
@@ -31,174 +31,289 @@ bool cancel()
     return false;
 }
 
+std::string shorter(const char* shorter)
+{
+    return std::string(shorter) + ",";
+}
+
+auto string_with_default(const char* defs) -> decltype(cxxopts::value<std::string>()->default_value(""))
+{
+    return cxxopts::value<std::string>()->default_value(defs);
+}
+
+auto double_with_default(const char* defs) -> decltype(cxxopts::value<double>()->default_value(""))
+{
+    return cxxopts::value<double>()->default_value(defs);
+}
+
+auto int_with_default(const char* defs) -> decltype(cxxopts::value<int>()->default_value(""))
+{
+    return cxxopts::value<int>()->default_value(defs);
+}
+
+std::vector<const char*> process_argv(int argc, const char** argv)
+{
+    std::vector<const char*> processed;
+    for (int i=0; i<argc; ++i)
+    {
+        processed.push_back(argv[i]);
+        #if defined(USE_SLASH_CLI_WINDOWS) && (defined(_WIN32) || defined(_WIN64))
+        // rpplace -- and - with /
```

Comment:
It is a complicated story here. 

We can basically replace `/` with `--` and pass it to the parsing library. But this doesn't help with shortened keywords because they have to be replaced with `-`. The library doesn't support anything about that with **'-' being hardcoded** (sic!).

I am in favour of just dropping the slash support here. 

@iglesias curious what you think.

Reply:
Other options might be to drop support of short keywords. The verbose style of WIndows commands would match that, too. We can also replace `-` style with CamelCase.

Reply:
I will get back :)

Reply:
Yeah... I think I see.

Among those two options, I would definitely choose dropping the slash support. It's subjective, though. When going through cxxopts README, I got the idea that one its main features is supporting long and short keywords.

Something involving some more work, but I think would help to replace `/` with `--` or `-`:
At the moment the long options are defined as `static const char*`s and then, each associated with the short version via calls to `either`.
Instead of the `const char*`s, I think we could use some kind of map from empty structs to string, or a set of strings if the tags aren't essential. At the moment of replacing, if it is in the map's values or in the set, then replace with `--`, detected as a long option, otherwise with `-`. Would that make sense, or would it clash with the error handling for unknown options? If the latter, then the mapping should associate to short options too and make two lookups.
In case it sounds good, I could work on this during this week. Or I could also let you to it and review, that's also fine of course. Any option is good :)

Reply:
Dropping it is then! Let me just commit it right away.

I think it is way too complicated to be worth spending your time.